device_backends/DummyBackend/include/DummyRegisterAccessor.h

```diff
@@ -173,6 +173,12 @@ namespace ChimeraTK {
           writeCallback);
     }
 
+    /// Set callback function which is called when the register is readed to (through the normal Device interface)
+    //void setReadCallback(const std::function<void()>& readCallback) {
```

Comment:
I think this part of code was prepared by Jan Timm in order to prepare some tests, but it was never done. The code is not used, I was not sure whether its needed so for time being I just commentd it, but I think it may be removed.